### DISCUSSION TITLE
footprint for EBYTE E49-900M20S

### DISCUSSION THREAD

**Discussion by opa-rudi** [tbnobody/OpenDTU#2275] [General]
Hi to all the solder smoke inhalers :),

does anybody have a footprint (KiCad) for the EBYTE E49-900M20S module? I don´t want to invent the wheel twice and I couldn´t find one out there.

greetings!

**Comment by stefan123t** [comment]
I made a Fritzing Part from a resized SVG.
Actually I extracted the foot print from the PDFs and images that Chengdu EByte and the RF chip CMT2300A manufacturers provide.

**Comment by ksjh** [comment]
Well, I created my own footprint in KiCad 8, intended for hand-soldering. I used it on a couple of PCBs. It works there without any problems.
Here you go: [E49-900M20S.kicad_mod](https://gist.github.com/ksjh/841ae6e447bc2f19904ea28d0dabdd39)
I also added the corresponding symbol: [e49-900m20s.kicad_sym](https://gist.github.com/ksjh/d0f269b402216571c9428ff3e4769be8)

This is how it looks on a PCB with a footprint for an ESP32-S3-Zero and pads for an I2C display:
![OpenDTU-S3](https://github.com/user-attachments/assets/fdad309c-418e-4814-82e1-00abb759f083)
Feel free to use it in your own projects.
